data/core.yaml

```diff
@@ -7,6 +7,7 @@ en:
     zoom_to: zoom to
```

Comment:
Usually when some change is done in core.yaml it's good to also push in the en.min.json  (you can do: npm run all) just to have this reflected immediately in en translation.